.github/workflows/github_pr.yml

```diff
@@ -61,12 +60,40 @@ jobs:
         sudo apt-get install libsodium-dev
         sudo apt-get install g++-9
 
+    - name: '[Windows] Install dependencies & setup environment'
+      if: runner.os == 'Windows'
+      shell: powershell
+      run: |
+        $url = "https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-msvc.zip"
+        $sha256hash = "C1D48D85C9361E350931FFE5067559CD7405A697C655D26955FB568D1084A5F4"
+        Write-Host ('Downloading {0} ...' -f $url)
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+        # See https://github.com/PowerShell/PowerShell/issues/2138
+        $ProgressPreference = 'SilentlyContinue'
+        New-Item -ItemType directory -Path ${{ github.workspace }}\lib\win64
+        Invoke-WebRequest -Uri $url -OutFile '${{ github.workspace }}\lib\libsodium.zip'
+        if ((Get-FileHash '${{ github.workspace }}\lib\libsodium.zip' -Algorithm "SHA256").Hash -ne $sha256hash) {
+          exit 1
+        }
+        Expand-Archive '${{ github.workspace }}\lib\libsodium.zip' -DestinationPath ${{ github.workspace }}\lib\
+        echo "::set-env name=LIB::${{ github.workspace }}\lib\libsodium\x64\Release\v140\static\;${{ github.workspace }}\submodules\d2sqlite3\lib\win64\;$LIB"
```

Comment:
Why were you using v140 here ? It should be v142: https://docs.microsoft.com/en-us/cpp/porting/binary-compat-2015-2017?view=vs-2019